From 2bffebcf9c125f5610b8fb660d42c3c5b9dfd4f0 Mon Sep 17 00:00:00 2001
From: Stefan Kost <ensonic@users.sf.net>
Date: Thu, 19 Aug 2010 14:45:54 +0000
Subject: fixxref: tweak the vim invocation

Use specific output-filename to ensure we get what we'll late use. Use "-u NONE"
instead of -u /dev/null.
Fixes #627223
---
diff --git a/gtkdoc-fixxref.in b/gtkdoc-fixxref.in
index 33a31b8..9f53852 100755
--- a/gtkdoc-fixxref.in
+++ b/gtkdoc-fixxref.in
@@ -464,7 +464,7 @@ sub HighlightSourceVim {
     close (NEWFILE);
 
     # format source
-    system "echo 'let html_number_lines=0|let html_use_css=1|let use_xhtml=1|syn on|e $temp_source_file|run! syntax/2html.vim|wa!|qa!' | @HIGHLIGHT@ -n -e -u /dev/null -T xterm >/dev/null";
+    system "echo 'let html_number_lines=0|let html_use_css=1|let use_xhtml=1|syn on|e $temp_source_file|run! syntax/2html.vim|w! $temp_source_file.html|qa!' | @HIGHLIGHT@ -n -e -u NONE -T xterm >/dev/null";
 
     my $highlighted_source;
     {
--
cgit v0.8.3.1
